client/src/components/ProjectCard.js

```diff
@@ -18,11 +19,9 @@ const IndustryLabel = styled('div')(({ theme }) => ({
 
 const styles = (theme) => ({
   card: {
-    display: 'flex',
-    flexDirection: 'column',
+    maxWidth: '650px',
```

Comment:
I didn't have defined widths while restyling on explore page as I let the grid above define the size of card

Reply:
Ah -- I just found that it stretched the pics too much. Think that should be done with height?

Reply:
just removed it